tests/keystore_test.rs

```diff
@@ -180,6 +180,38 @@ fn should_create_keyfile() {
 #[test]
 fn should_use_correct_filename() {}
 
+#[test]
+fn should_search_by_address() {
+    let addr = "0x0047201aed0b69875b24b614dda0270bcd9f11cc"
+        .parse::<emerald::Address>()
+        .unwrap();
+
+    let res = search_by_address(&keystore_path(), &addr);
+    assert!(res.is_some());
+
+    let kf = res.unwrap();
+    assert_eq!(kf.cipher_text,
```

Comment:
No i mean it's anough just to proof `uuid` from `KeyFile` to be sure that all right.